x-pack/plugins/security_solution/public/overview/components/detection_response/cases_table/cases_table.test.tsx

```diff
@@ -76,7 +76,7 @@ describe('CasesTable', () => {
     mockUseCaseItemsReturn({ isLoading: false });
     const { getByText } = renderComponent();
 
-    expect(getByText('Updated now')).toBeInTheDocument();
+    expect(getByText(/Updated/)).toBeInTheDocument();
```

Comment:
These tests are async, and  caused flaky tests with 'now' not being in the doc as time passed before the check. So instead we're just checking that this part of the text is there.